share/spice/playing_cards/content.handlebars

```diff
@@ -0,0 +1,5 @@
+<div>
```

Comment:
Ah sorry, you also need to set `item_detail: false` because the `movies` template group also specifies an `item_detail` template, which is what it's trying to use.